overlay-app/src/css/Scoreboard.css

```diff
@@ -1,82 +1,144 @@
-.scoreboard {
+.scoreboard-wrapper {
```

Comment:
Added a wrapper here to fill the width of the screen, so I could `margin: 0 auto` to center the scoreboard no matter the scoreboard width